### ISSUE TITLE
Something changed in VS 2022 Preview 3 and VS 2019 16.11 that breaks VB WinForms Apps

### ISSUE THREAD

**Issue by paul1956** [dotnet/sdk#19735]
**Visual Studio Version**:
VS 2022 Preview 3 and VS 2019 16.11

**Summary**:
System and Microsoft.VisualBasic as well are many other libraries now require an explicate Import

**Steps to Reproduce**:
any existing WInforms application now requires explicate imports in designer file
```
Imports System.Diagnostics
Imports System.Runtime
Imports Microsoft.VisualBasic

```

**Expected Behavior**:
Project should just work

**Actual Behavior**:
1000's of errors
**User Impact**:
lots of manual editing to add all the missing imports, including adding "System" to Project file Include section.

**Comment by dsplaisted** [comment]
Yes, this looks like the same thing.  If someone can share a [binlog](https://aka.ms/binlog) of a build with this issue that would help me look into it.

**Comment by Nukepayload2** [comment]
Probably caused by https://github.com/dotnet/sdk/commit/f2cc77b60cdc2a4e8b393adf7b855b478ad79671#diff-5dfdd31992820273a25b399353a85ec6de6bdd620424a8adab3238c6ffd5df38R13    .
I don't know why they added ` and '$(TargetFrameworkIdentifier)' == '.NETFramework'` .

**Comment by dsplaisted** [comment]
> Probably caused by [f2cc77b#diff-5dfdd31992820273a25b399353a85ec6de6bdd620424a8adab3238c6ffd5df38R13](https://github.com/dotnet/sdk/commit/f2cc77b60cdc2a4e8b393adf7b855b478ad79671#diff-5dfdd31992820273a25b399353a85ec6de6bdd620424a8adab3238c6ffd5df38R13) .
> I don't know why they added ` and '$(TargetFrameworkIdentifier)' == '.NETFramework'` .

I don't think that's the issue.  There is one `ItemGroup` for `'$(TargetFrameworkIdentifier)' == '.NETFramework'`, and another one for `'$(DisableImplicitNamespaceImports)' != 'true' and '$(_IsNETCoreOrNETStandard)' == 'true'`.
